skimage/filters/thresholding.py

```diff
@@ -1104,16 +1105,24 @@ def threshold_multiotsu(image, classes=3, nbins=256):
 
     Returns
     -------
-    idx_thresh : array
+    thresh_vals : array
         Array containing the threshold values for the desired classes.
 
+    Raises
+    ------
+    ValueError
+         If ``image`` contains less grayscale value then requested
```

Comment:
`If ``image`` contains less grayscale values than the desired number of classes.`